content/en/real_user_monitoring/application_monitoring/ios/integrated_libraries.md

```diff
@@ -41,7 +44,34 @@ import DatadogRUM
 
 URLSessionInstrumentation.enable(with: .init(delegateClass: Apollo.URLSessionClient.self))
 ```
-For additional information on sampling rate, distributed tracing, and adding custom attributes to tracked RUM resources, refer to [Advanced Configuration > Automatically track network requests][4].
+For additional information on sampling rate, distributed tracing, and adding custom 
+attributes to tracked RUM resources, refer to [Advanced Configuration > Automatically track 
+network requests][4].
+
+3. Add the [Datadog Apollo interceptor][8] package to your `Package.swift` file:
+
+```swift
+dependencies: [
+    .package(url: "https://github.com/DataDog/dd-sdk-ios-apollo-interceptor", .upToNextMajor(from: "1.0.0"))
+]
+```
+
+4. Add the Datadog interceptor to your Apollo Client setup:
+
+```swift
+import Apollo
+import DatadogApollo
+
+class CustomInterceptorProvider: DefaultInterceptorProvider {
+    override func interceptors<Operation: GraphQLOperation>(for operation: Operation) -> [ApolloInterceptor] {
+        var interceptors = super.interceptors(for: operation)
+        interceptors.insert(DatadogApollo.createInterceptor(), at: 0)
+        return interceptors
+    }
+}
+```
```

Comment:
````suggestion
   ```swift
   import Apollo
   import DatadogApollo

   class CustomInterceptorProvider: DefaultInterceptorProvider {
       override func interceptors<Operation: GraphQLOperation>(for operation: Operation) -> [ApolloInterceptor] {
           var interceptors = super.interceptors(for: operation)
           interceptors.insert(DatadogApollo.createInterceptor(), at: 0)
           return interceptors
       }
   }
   ```
````